### PR TITLE
MGMT-2138 Added asisted-iso-create rhcos image download back

### PR DESCRIPTION
**Pull request by mvazquezc**
Adds the RHCOS image download back to the Dockerfile for livecd iso creation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,37 @@ skipper make update && kubectl get pod --namespace assisted-installer -o name |
...
+   2. Fedora CoreOS: https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable￼
+2. Build the new iso generator image
+
+   ~~~sh
+￼  # Example with RHCOS
+￼  BASE_OS_IMAGE=https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/rhcos-4.6.0-0.nightly-2020-08-26-093617-x86_64-live.x86_64.iso make build-assisted-iso-generator-image
+￼  ~~~
+
+
+## Update Discovery Image base OS
```
Comment (rwsu) [inline README.md:105]: Lines 105-118 repeats above.

**Review by rwsu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mvazquezc) [reply README.md]: Fixed!

**Review by avishayt — APPROVED** [review]

**Comment by mvazquezc** [comment]
Closed in favor of: #486

**Pull request closed without merging**